Youtube API v3 + video_duration + more

Daugilas opened the pull request:
Main fix is to make it work with Youtube API v3: https://developers.google.com/youtube/v3/getting-started#before-you-start

Added some new features: display video duration, define/set video title, hide info/title bar, and few more advanced options described in readme.md

Daugilas committed:
Display duration option

Daugilas committed:
readme.md adding info on default parameters

Daugilas committed:
update to work with Youtube API v3

Daugilas committed:
trivial: optimize PT to Time conversion

Daugilas committed:
Allow override of video title via div title attr

Also via data-title attribute.
When title is set and duration is not requested - no xhr request for
that video is needed!

Daugilas committed:
Add option to show/hide the title

According to https://github.com/tylerpearson/lazyYT/pull/39/files by
@Jmeyering

Daugilas committed:
trivial: formatting

Daugilas committed:
trivial: version 1.1.0

Daugilas committed:
enable to customize class added after video loads

issue: https://github.com/tylerpearson/lazyYT/issues/28

Daugilas committed:
Update readme.md

to work with Youtube API v3

Daugilas committed:
trivial: readme.md

Daugilas committed:
trivial: readme.md

Daugilas committed:
trivial: readme.md

Daugilas committed:
Add Youtube URL parameters option

as requestod by @mgrn0  https://github.com/tylerpearson/lazyYT/issues/44

Daugilas committed:
version update to v1.1.1

Daugilas commented:
@tylerpearson what about merging this PR? Or giving me the access to this repo to do that?

tylerpearson commented:
Hey, sorry about that -- this looks awesome! Can you resolve the conflicts in the PR? Then I can merge it in.

Daugilas committed:
add callback functionality

request by @sachbearbeiter
https://github.com/tylerpearson/lazyYT/issues/21

Daugilas committed:
add changelog.md

Daugilas committed:
New YouTube UI

Also pointed out here https://github.com/tylerpearson/lazyYT/issues/46
by @jaskaran27

Daugilas committed:
Version to 1.2

Daugilas committed:
Play button CSS fix

Found by @rotemberger here:
https://github.com/tylerpearson/lazyYT/issues/47

Commit:
The callback fuction is called after inserting the iframe

Daugilas committed:
Merge pull request #5 from abraxas-medien/feature/usefull-callback

The callback fuction is called after inserting the iframe

vegantech committed:
fix minor typo in readme

dimaip commented:
Any updates on this? Doesn't work with old API any longer

Daugilas committed:
Merge pull request #9 from vegantech/patch-1

fix minor typo in readme

Daugilas committed:
bugfix: added ytp-spinner and thumbnail validation

regarding @adamjcsmith issue:
https://github.com/tylerpearson/lazyYT/issues/49

Daugilas commented:
@dimaip its solved on this fork: https://github.com/Daugilas/lazyYT

Daugilas committed:
trivial: fix image loading for FireFox

Daugilas committed:
trivial: irrelevand changes

Daugilas committed:
trivial: add Fork me on GitHub ribbon

thanks to: https://codepro8.github.io/css-fork-on-github-ribbon/

Daugilas committed:
trivial: add Fork me on GitHub ribbon

thanks to: https://codepo8.github.io/css-fork-on-github-ribbon/

Daugilas committed:
Version to 1.3.0

Brugman committed:
Use https iframe url.

Daugilas committed:
Merge pull request #12 from Brugman/feature/remove-protocol-relative-urls

remove protocol relative urls
